### ISSUE TITLE
Fine tune accuracy

### ISSUE THREAD

**Issue by ashwinijoshigithub** [neokt/car-damage-detective#8]
I am trying out this code on my own data. However, my train_top_model() gives very good results with around 84% of accuracy. But while finetuning, my accuracy on validation doesn't change at all even after increasing the size of the data. I also tried reducing learning rate and keeping it very low but doesn't help. Any idea why this happens? Please help @neokt

**Comment by hareemfatima10** [comment]
@ashwinijoshigithub Hi. I am trying to run this code with the dataset given and since im fairly new to all this I am having a hard time running it (many errors and issues). I am also not sure if I am on the right track. Would you be able to explain a little on how to see this in action maybe with the already trained model?
